packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/routing/scripts/cloud-sdk-init.component.ts

```diff
@@ -15,7 +15,7 @@ export class CloudSdkInitComponent implements OnInit {
   constructor() {}
 
   ngOnInit(): void {
-    if (!isServer) {
+    if (!isServer()) {
```

Comment:
Would be nice if you can update this condition and don't initialize CloudSDK if it's "Development" mode